DESCRIPTION

```diff
@@ -35,7 +35,7 @@ Depends:
     R (>= 4.1)
 Imports:
     cli (>= 3.0.0),
-    dplyr (>= 1.0.5),
+    dplyr (>= 1.1.1),
```

Comment:
@bundfussr and @manciniedoardo  well huh that is interesting catch that admiral and admiraldev have different dplyr versions.  eck!!